packages/main/src/themes/PopupsCommon.css

```diff
@@ -43,8 +43,6 @@
 .ui5-popup-footer-root,
 :host([header-text]) .ui5-popup-header-text {
     margin: 0;
-    font-size: 1rem;
-    font-family: "72override", var(--_ui5_popup_header_font_family);
```

Comment:
Looks like this is needed. Dialog's title now appears in `Times New Roman` with 1.5rem font size

Reply:
ui5-dialog and ui5-popover templates are adapted as well